es/webapp/src/main/webapp/i18n/all_es.properties

```diff
@@ -946,3 +946,5 @@ label.dateTimeWithPrecision.day_capitalized = Día
 label.dateTimeWithPrecision.hour_capitalized = Hora
 label.dateTimeWithPrecision.minutes_capitalized = Minutas
 label.dateTimeWithPrecision.seconds_capitalized = Segundos
+
+new_entry_for = "{0}" entry for {1}
```

Comment:
Is there a reason why this line is in English, even though this properties file is for Spanish i18n?

Reply:
Fixed that